src/main/java/lotto/LottoRank.java

```diff
@@ -0,0 +1,44 @@
+package lotto;
+
+import java.util.Arrays;
+
+public enum LottoRank {
+    FIRST(6, 2_000_000_000),
+    SECOND(5, 30_000_000),
+    THIRD(5, 1_500_000),
+    FOURTH(4, 50_000),
+    FIFTH(3, 5_000),
+    OUT_OF_RANK(0, 0);
+
+    private static final int FIVE = 5;
+
+    private int countOfMatch;
+    private int price;
+
+    LottoRank(int countOfMatch, int price) {
+        this.countOfMatch = countOfMatch;
+        this.price = price;
+    }
+
+    public int getCountOfMatch() {
+        return countOfMatch;
+    }
+
+    public int getPrice() {
+        return price;
+    }
+
+    public static LottoRank valueOf(int countOfMatch, boolean bonusMatch) {
+        if (bonusMatch && countOfMatch == FIVE) {
+            return SECOND;
+        }
+        if (!bonusMatch && countOfMatch == FIVE) {
+            return THIRD;
+        }
+        return Arrays.stream(LottoRank.values())
+                .filter(lottoRankEnum -> lottoRankEnum.getCountOfMatch() == countOfMatch)
+                .findFirst()
```

Comment:
이런 부분은 조금 아쉬운 부분이에요. 
`lottoRankEnum.getCountOfMatch()` getter를 사용할 필요가 전혀 없습니다. 
클래스 내부에서는 해당 인스턴스의 필드에 자유롭게 접근이 가능합니다. 

```suggestion
                .filter(lottoRankEnum -> lottoRankEnum.countOfMatch == countOfMatch)
                .findFirst()
```

Reply:
같은 클래스 내부의 멤버변수를 적어주는게 맞네요!
이런 부분들은 신경 써야겠습니다